components/combobox/combobox.vue

```diff
@@ -237,7 +270,7 @@ export default {
         // The list has to be positioned relatively so that the auto-scroll can
         // calculate the correct offset for the list items.
         class: 'd-ps-relative',
-        'aria-label': this.listAriaLabel,
+        'aria-label': this.label,
```

Comment:
You can remove this the listbox doesn't need an aria label.

Reply:
I can see an error on a11y test now related to `role="listbox"` I believe its because this update.
Its seems its necessary to have this `aria-label` or... Im missing sth?